### PR TITLE
Add support for "Description position" 

### PR DESCRIPTION
**Pull request by stevenmusumeche**
Follow up from https://github.com/mizdra/eslint-interactive/issues/285

### CODE REVIEW

**Comment by mizdra** [comment]
Thanks! I will review and add tests.

File: src/__snapshots__/core.test.ts.snap [hunk 1]
```diff
@@ -1680,7 +1680,7 @@ exports[`Core printSummaryOfResults 1`] = `
 ╟─────────────────────────────┼───────┼─────────┼────────────┼─────────────────╢
 ║ ]8;;https://eslint.org/docs/rules/semisemi]8;;                        │ [31m[1m14[39m[22m    │ [31m[1m1[39m[22m       │ [31m[1m15[39m[22m         │ 0               ║
 ╟─────────────────────────────┼───────┼─────────┼────────────┼─────────────────╢
-║ ]8;;https://github.com/import-js/eslint-plugin-import/blob/v2.26.0/docs/rules/order.mdimport/order]8;;                │ [31m[1m3[39m[22m     │ 0       │ [31m[1m3[39m[22m          │ 0               ║
+║ ]8;;https://github.com/import-js/eslint-plugin-import/blob/v2.27.5/docs/rules/order.mdimport/order]8;;                │ [31m[1m3[39m[22m     │ 0       │ [31m[1m3[39m[22m          │ 0               ║
```
Comment (mizdra) [inline src/__snapshots__/core.test.ts.snap:1683]: Since this PR does not update the package version, it is wrong to update this snapshot.

I think this test is failing because you have not installed the version of the package based on pnpm-lock.yaml. Are you installing the package with `npm install` or `yarn install` instead of `pnpm instaii`?

Comment (smusumeche) [reply src/__snapshots__/core.test.ts.snap]: I used npm, not pnpm, so that explains it.  Thanks for fixing.

**Comment by mizdra** [comment]
I have added several commits to fix the edge case issue. I also added some tests. I believe the PR is now of mergeable quality.

---

At first I thought this feature would be easy to implement. However, it turned out to be difficult to implement because of the number of edge cases. This was an error in judgment on my part. Nevertheless, thanks to @stevenmusumeche for boldly implementing it! 👍

**Review by mizdra — APPROVED** [review]
LGTM!
